.travis.yml

```
@@ -30,6 +37,30 @@ jobs:
 #  - stage: test
 #    dist: bionic
 #    python: 3.8
+  - stage: experimental
+    os: windows
+    name: "Python 3.6.7 on Windows"
+    language: sh
```

```suggestion
    language: bash
```

Recommended since our pandas install uses bash syntax (and travis docs say our options are `shell` (vs sh) and `bash)